scripts/generate-app-config/generatePagesConfig.js

```diff
@@ -136,6 +136,104 @@ const buildPageConfig = (filePaths, generatePagesOptions, namespace) => (
   }, {})
 );
 
+const getScreenshotPatterns = generatePagesOptions => (
```

Comment:
Could this simply be an array of patterns or do we need to filter on the search Patterns? We are globing below which makes me think this could be simplified?